api/Controllers/MiniUrl.cs

```diff
@@ -48,7 +40,8 @@ public ActionResult<MinifyResponse> Post([FromBody] MinifyRequest body)
         string urlId = Guid.NewGuid().ToString();
         urlId = urlId.Substring(0, 8);
```

Comment:
I think it is good idea to move ID generation into RedisActions service itself

Reply:
done